src/main/kotlin/org/malachite/estella/config/WebSecurityConfig.kt

```diff
@@ -55,4 +66,28 @@ class WebSecurityConfig(val jwtFilter: JwtSecurityFilter, val userDetailsService
 
     @Bean
     fun passwordEncoder(): PasswordEncoder = BCryptPasswordEncoder()
+
+    companion object {
+        private fun apiKey(): ApiKey = ApiKey("JWT", EStellaHeaders.jwtToken, "header")
+        private fun securityContext(): SecurityContext {
+            return SecurityContext.builder().securityReferences(defaultAuth()).build()
+        }
+        private fun defaultAuth(): List<SecurityReference> {
+            val authorizationScope = AuthorizationScope("global", "accessEverything")
+            val authorizationScopes = arrayOf(authorizationScope)
+            return listOf(SecurityReference("JWT", authorizationScopes))
+        }
+    }
+
+    @Bean
+    fun api(): Docket {
+        return Docket(DocumentationType.SWAGGER_2)
+            .apiInfo(ApiInfo.DEFAULT)
+            .securityContexts(listOf(securityContext()))
+            .securitySchemes(listOf(apiKey()))
+            .select()
+            .apis(RequestHandlerSelectors.any())
+            .paths(PathSelectors.any())
+            .build()
+    }
```

Comment:
Z racji, że nie mam pojęcia, co się tutaj dzieje, ufam, że to działa i jest sprawdzone w każdym możliwym przypadku

Reply:
To jest konfig, który pozwala swaggerowi odpowiednio wrzucać token do requestów